security/tls.go

```diff
@@ -150,3 +165,29 @@ func LoadInsecureClientTLSConfig() *tls.Config {
 		InsecureSkipVerify: true,
 	}
 }
+
+// LogRequestCertificates examines a http request and logs a summary of the TLS config.
+func LogRequestCertificates(r *http.Request) {
```

Comment:
all of this per-request-even-if-everything-is-ok stuff could be better of in `V(2)` (or higher). We've recently tried to denoise `V(1)` somewhat.